superset-frontend/package-lock.json

```diff
@@ -7689,28 +7670,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
+              "resolved": false,
```

Comment:
I don't understand why a clean install `npm ci` would update `resolved` to `false`.  A quick google suggests `npm >= 5.0.0` will [simply ignore this field](https://stackoverflow.com/questions/53814967/what-is-the-point-of-having-resolved-url-in-package-lock-json). Did someone updated `pcakge-lock.json` with a lower version of `npm`?

Reply:
so weird. I use `npm@6`

Reply:
@kristw Do you get the same package-lock as me when you install?

Reply:
Let me try

Reply:
yeah. did `npm install` also see `false`

Reply:
i'm on 6.14.2, so not sure here